### ISSUE TITLE
[Form] Image data value not valid

### ISSUE THREAD

**Issue by Snowbaha** [symfony/symfony#46785]
### Symfony version(s) affected

"symfony/form": "5.4.9" and "5.4.10"

### Description

I updated all my project composer on the last 5.4.* but I had to fix the 5.4.8 symfony form. The project is runnig on PHP 8.1.

When I do a POST request with my symfony API with a data image, it is not working whereas it was working before : 
![image](https://user-images.githubusercontent.com/3284722/175894065-f5b11380-7007-4e36-9059-79a7ae366046.png)

Here is the part of my entity: 
```php
class SliderItem
{
    use TimestampableEntity;

    /**
     * @var string
     *
     * @ORM\Column(name="image", type="string", length=255, nullable=true)
     *
     * @Assert\File(
     *     maxSize = "5M",
     *     mimeTypes = {"image/jpeg", "image/jpg", "image/gif", "image/png"},
     *     mimeTypesMessage = "Le fichier choisi ne correspond pas à un fichier valide",
     *     notFoundMessage = "Le fichier n'a pas été trouvé sur le disque",
     *     uploadErrorMessage = "Erreur dans l'upload du fichier"
     * )
     *
     * @Groups({"slider_item","slider_items","slider", "slider_public"})
     */
    private $image;

//....
```

and the form builder : 
```php
$builder
            ->add('image')
            ->add('caption')
            ->add('content')
            ->add('position', IntegerType::class, ['required' => false])
            ->add('bgPosition')
            ->add('active')
        ;
```

Maybe someting change with chlidren data? `children[image]`

### How to reproduce

Here the raw request :
```
{
    "bgPosition": "centre center",
    "caption": "sliderTest",
    "image": {
    	"name" : "test image.png",
		"originaleExtension" : "png",
		"extension" : "png",
		"type" : "image/png",
		"size" : 43510,
    	"data" : "iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAADM0lEQVRIic2VS2icVRTHf/fe70tmMplHJ8l0ZpIOFhNTG9sYUrCitAv3iloVi0EpKPWBpYgr6ULcFCxKpaituEpBd25d+SBKtSgVrBYitbYNmrTziJ04k3zfd+9xMcFEWu00UOiBw33+z++e++AqEeFmmr6p0W85QPqlnx7N7Dvz0I1oVLtn0HPgx97GtDqjfRPEh9WW8oFN1XZ0bWfQaLoPd+1M9j7xQKbYiORou7q2AOlXfng13aUfHB1P8tiOJAPZzl3Z16dfbkd73S1K7z91n9Hq8/6NWd8kDCNJx9JfEZ+dj5a0UTvKrw2eXHMGmX3fZ8XayUIx6dfjcf6Yd5w63wQjjOT9ztAx2Xfw18yaAVHoPujr6droClkaoUOAat1y9lLAhoxmIO3dEQrvrgmQfP7k/mTceyS1qUDNKgyggCB0XPoz4mI1YGvBkPD1k71v/vbCDQGSe7/d7uEOFkf7qfhxtFt1TiI0Fh0z1Yj5RsR4v48TDuXeujDeFiD13ImMC6LJ0uZ8Rz3fg40satW4OMAJ9YblXDmiQ1s29/nx0DG5/vBM6rqAKLBHiwOpQUZKNAK5eoIIiOCsUFuwnL0cUkpDPmnujIQj/wvoembqxUy3/3h6+xA10ehrXWERxLUgYei4fMVysRayNWeIeXoid+T3Z68JSDw9ta1D5FDx3tupJGJoe3XslQxaBSI0A8dMzXKlGTKWNzjU2/n3Z0f/Beie+DIVLobHS2MDsfqGXlzwH8EBnIBzy95qLzQt5yoRMe0YXGcSTcvxwrG57n8AzWb03j2juWH/7ttYWFpe6Cp3suLWCdYJzgnOOZwTrBWqdccv5YCdJZ+nhhN3hb4+DOCtm/hi7+771+/e8/AQb8xqGgKeakV2ohCrcFbhnMaKJohpiATdoVGeBk+jPAUeBFYYy3kMEePnr8I9W47VvlaffDO3cKESJD7+rsLcIlgUSiuU0aAUaIVSqvXKtG5tvlZoo2HZRevWfKPJpQxuUTF9QhFbNA3vo6nZdz49PV+oh0LnKqEyGuWZVt1b7vPMSp8RMIIygjaCLJfVikN1aLxtimZZZtv+cNZqt9afvBb7G3wGayi0bz8+AAAAAElFTkSuQmCC"
    }
}
```
when I post in `{{domain}}/cms/sliders/1/items`

**Comment by xabbuh** [comment]
Can you create a small example application that allows to reproduce your issue?

**Comment by xabbuh** [comment]
I am going to close here for now due to the lack of feedback. Please let us know when you have more information and we can consider to reopen.

**Comment by Snowbaha** [comment]
Yes sorry I don't have enough  time to build a special app for the example. For now I fixed the version to 5.4.8.

**Comment by xabbuh** [comment]
No worries, feel free to come back with an example once you have the time for it and we can consider to reopen then.
